processor/src/main/java/org/mapstruct/ap/internal/util/Strings.java

```diff
@@ -89,11 +86,11 @@ private Strings() {
     }
 
     public static String capitalize(String string) {
-        return string == null ? null : string.substring( 0, 1 ).toUpperCase() + string.substring( 1 );
+        return string == null ? null : string.substring( 0, 1 ).toUpperCase( Locale.ENGLISH ) + string.substring( 1 );
```

Comment:
The current Javadoc of `toUpperCase` suggests using `Locale.ROOT` instead of `Locale.ENGLISH`...